Makefile

```diff
@@ -43,9 +43,13 @@ TEST_LDFLAGS=-ldflags "-X github.com/cilium/cilium/pkg/kvstore.consulDummyAddres
 	-X github.com/cilium/cilium/pkg/kvstore.etcdDummyAddress=http://etcd:4002 \
 	-X github.com/cilium/cilium/pkg/testutils.CiliumRootDir=$(ROOT_DIR) \
 	-X github.com/cilium/cilium/pkg/datapath.DatapathSHA=1234567890abcdef7890"
+
+# If you want to change the logging level of all unit tests below, you can change
+# the value of github.com/cilium/cilium/pkg/logging.DefaultLogLevelStr.
 TEST_UNITTEST_LDFLAGS= -ldflags "-X github.com/cilium/cilium/pkg/kvstore.consulDummyConfigFile=/tmp/cilium-consul-certs/cilium-consul.yaml \
 	-X github.com/cilium/cilium/pkg/testutils.CiliumRootDir=$(ROOT_DIR) \
-	-X github.com/cilium/cilium/pkg/datapath.DatapathSHA=1234567890abcdef7890"
+	-X github.com/cilium/cilium/pkg/datapath.DatapathSHA=1234567890abcdef7890 \
+	-X github.com/cilium/cilium/pkg/logging.DefaultLogLevelStr=error"
```

Comment:
It would probably be better if we would use a variable to set this value. This variable would be set on line 32 by checking if `LOG_LEVEL` is set. If it isn't, should default to `error`.

Reply:
Done.